tex/chapters/survey/main.tex

```diff
@@ -1,6 +1,71 @@
 \chapter{Data collection}
 
 \section{Background on survey design}
+
+%cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
+%cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/
+
+A survey is a method of data collection with the purpose of obtaining quantitative information regarding opinions and 
+experiences of the respondents. Surveys could be used to investigate, for example, a product for which a company would like 
+consumer feedback, or how a disease impacts patients in their daily lives. This form of investigation is useful for gathering a
+large amount of data that can be easily quantified and analysed using statistical methods and tests. 
```

Comment:
Can you add a sentence or two with some examples of "famous" survey led results.